### PR TITLE
Add car details screen

### PR DESCRIPTION
**Pull request by The0mikkel**
This updates solves #18 by adding a car details screen to the application.

The screen holds a single image of the car, as well as the name and a button forward. 

This screen may be refactored in the future, as the same layout has to be used for multiple frames.

### CODE REVIEW

**Review by OthelloEngineer — APPROVED** [review]
Looking good captain. 
We should probably have had an Issue that looks into informative error handling like the NotFound component. Appreciate the addition though.

File: assets/right-arrow.png
Comment (jakob753951) [inline assets/right-arrow.png:1]: Not a fan of this being a png, but i don't know if svg is more difficult.

**Review by jakob753951 — APPROVED** [review]
LGTM

**Review by Ashvikan — APPROVED** [review]

File: assets/right-arrow.png
Comment (The0mikkel) [reply assets/right-arrow.png:1]: I actually don't know or haven't tried.

Comment (The0mikkel) [reply assets/right-arrow.png:1]: Maybe, we should do a complete redo of images at one point.

**Pull request merged**